javascript/javascript1/week2/weather.js

```diff
@@ -0,0 +1,20 @@
+function clothesToWear(temp) {
+    if (temp <= 0) {
+        alert("Please wear a  Bomber jacket as snow fall is continuing here in Demark!!");
+    }
+    else if (temp <= 10) {
+        alert("Please wear a proper winter jacket it's too cold here in Demark!!");
+    }
+    else if (temp > 10 && temp <= 15) {
+        alert("Please wear a  Light jacket as its Autumn here in Demark!!");
+    }
+    else if (temp <= 25 && temp > 15) {
+        alert("Please wear a  sweatshirt as its windy  here in Demark!!");
+    }
+    else
+        alert("Oh temerature is rising  lets try some summer cool clothings!");
+
+}
+
+const temperature = prompt("What is todays temperature ?");
```

Comment:
try always use console.log to show a message . because alert is only for alert . This function has one bug which is when I input my own name instead of temperature , it suggest me wearing summer clothes. In the question they don't ask about data input so there is no need to use prompt() which creates a little bug. Also it might be a good exercise to practice switch() method.

Reply:
Okay,will fix this,thank you.